driver/razerkbd_driver.c

```diff
@@ -1040,9 +1040,9 @@ static ssize_t razer_attr_read_set_logo(struct device *dev, struct device_attrib
     struct usb_interface *intf = to_usb_interface(dev->parent);
     struct usb_device *usb_dev = interface_to_usbdev(intf);
     struct razer_report report = razer_chroma_standard_get_led_effect(VARSTORE, LOGO_LED);
-    struct razer_report response;  
```

Comment:
and here you left 2 spaces at the end of the line ^^

Reply:
ah. i'll just go blame gedit for both. i should really install clion at some point.